lms/templates/seq_module.html

```diff
@@ -44,10 +44,11 @@
      ${item['content'] | h}
   </div>
   % endfor
-  <div id="seq_content" role="tabpanel"></div>
+  <div id="sr-is-focusable" tabindex="-1"></div>
```

Comment:
NIT: should we separate this class into two classes: ``sr`` and ``is-focusable``?